extensions/guacamole-auth-ldap/src/main/java/org/glyptodon/guacamole/auth/ldap/connection/ConnectionService.java

```diff
@@ -109,11 +109,38 @@
             // possibly be null
             assert(userDN != null);
 
-            // Find all Guacamole connections for the given user
+            // If group base DN is specified search for user groups
+            String groupBaseDN = confService.getGroupBaseDN();
+            // Create a separate filter for user groups
```

Comment:
To clarify a bit, it's odd to see something like:

```java
// If group base DN is specified search for user groups
String groupBaseDN = confService.getGroupBaseDN();
// Create a separate filter for user groups
StringBuilder userRoleGroupFilter = new StringBuilder();
if (groupBaseDN != null) {
```

Normally, you would expect a blank line to separate two logical paragraphs of code like this.